src/test/java/hudson/plugins/ec2/SpotResubmitTest.java

```diff
@@ -0,0 +1,146 @@
+package hudson.plugins.ec2;
+
+import com.amazonaws.services.ec2.model.SpotInstanceRequest;
+import com.amazonaws.services.ec2.model.SpotInstanceStatus;
+import hudson.model.Action;
+import hudson.model.Actionable;
+import hudson.model.Descriptor;
+import hudson.model.Executor;
+import hudson.model.Queue;
+import hudson.model.Result;
+import hudson.model.TaskListener;
+import hudson.plugins.ec2.ssh.EC2UnixLauncher;
+import hudson.slaves.OfflineCause;
+import hudson.slaves.SlaveComputer;
+import jenkins.model.Jenkins;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import hudson.model.queue.SubTask;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Logger;
+
+import static org.mockito.Mockito.*;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.whenNew;
+
+@RunWith(PowerMockRunner.class)
```

Comment:
Adding
`@PowerMockIgnore({"javax.crypto.*", "org.hamcrest.*", "javax.net.ssl.*", "com.sun.org.apache.xerces.*", "javax.xml.*", "org.xml.*"})`
as seen AmazonEC2CloudUnitTest.java and EC2StepTest.java in might resolve the test failure

Reply:
oh hey I actually forgot about this PR... let me try that, thanks!